apps/scraper/tasks.py

```diff
@@ -0,0 +1,9 @@
+from core.celery import app
+from apps.scraper.util_for_scraping import craulers
+
+
+@app.task
+def crauler(sites):
+    for site in sites:
+        getattr(craulers, site)()
+    print('Done')
```

Comment:
Would be cool if the task return some result.

Reply:
I do not think it is necessary.

Reply:
How are you going to determine that crawling had finished? especially if you don't have access to stdout